golang/examples/hello/client/main.go

```diff
@@ -72,22 +68,24 @@ func main() {
 
 	call, err := ch.BeginCall(ctx, *peerAddr, *serviceName, *operationName)
 	if err != nil {
-		log.Errorf("Could not begin call to %s.%s@%s: %v", *serviceName, *operationName, *peerAddr, err)
+		log.Errorf("Could not begin call to %v.%v@%v: %v", *serviceName, *operationName, *peerAddr, err)
+		log.Errorf("Is the server running?")
+
 		os.Exit(-1)
 	}
 
-	if err := call.WriteArg2(asArgument(*arg2)); err != nil {
+	if err = call.WriteArg2(asArgument(*arg2)); err != nil {
 		log.Errorf("Could not write arg2: %v", err)
 		os.Exit(-1)
 	}
 
-	if err := call.WriteArg3(asArgument(*arg3)); err != nil {
+	if err = call.WriteArg3(asArgument(*arg3)); err != nil {
```

Comment:
I'll change it back, but variables should only be initialized once.  I'm actually surprised the compiler doesn't bitch here, it should.